esx_service/utils/vmdk_utils.py

```diff
@@ -74,7 +74,8 @@ def init_datastoreCache():
             continue
         datastores.append((datastore.info.name,
                            os.path.split(datastore.info.url)[1],
```

Comment:
I doubt this statement may not really work. I tried to run this statement with a real datastore url, it always return empty:

>>> import os
>>> url =os.path.split("ds:///vmfs/volumes/vsan:525ab51e793147fd-c0f242a06e800fc2/")[1] 
>>> print(url)

If this field is incorrect and it's not being used anywhere, we should just get rid of it.